documentation/modules/oauth/proc-oauth-authorization-keycloak-example.adoc

```diff
@@ -256,14 +277,34 @@ keytool -keystore /tmp/truststore.p12 -storetype pkcs12 -alias sso -storepass $S
 KAFKA_HOST_PORT=my-cluster-kafka-bootstrap:9093
 STOREPASS=storepass
 
-echo "Q" | openssl s_client -showcerts -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.crt
+echo "Q" | openssl s_client -showcerts -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.pem
 ----
 
+If the client accesses the Kafka cluster over the Ingress or the OpenShift Route the Kafka bootstrap hostname will be different, and you may need to specify it as an extra argument to the `openssl` command in order to get the certificate containing the proper SNA records:
+[source,shell]
+----
+KAFKA_BOOTSTRAP_EXTERNAL=kafka-bootstrap-addr.somedomain.ext
+STOREPASS=storepass
+
+echo "Q" | openssl s_client -showcerts -servername $KAFKA_BOOTSTRAP_EXTERNAL -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.pem
+----
++
+The obtained `.pem` file is usually not one single certificate, but a certificate chain. You only have to provide the top-most issuer CA which is listed as last in the `/tmp/my-cluster-kafka.pem`.
+You can extract it manually or using the following command:
++
+[source,shell,subs="+quotes"]
+----
+split -p "-----BEGIN CERTIFICATE-----" /tmp/my-cluster-kafka.pem kafka-
+for f in $(ls kafka-*); do mv $f $f.pem; done
+cp $(ls kafka-* | sort -r | head -n 1) my-cluster-kafka-ca.crt
+----
+
+
 . Add the certificate for the Kafka broker to the truststore.
```

Comment:
This shows as 1 in the [file view](https://github.com/strimzi/strimzi-kafka-operator/blob/9d3fc4739d80947244614dbe0e3418d47963d3a2/documentation/modules/oauth/proc-oauth-authorization-keycloak-example.adoc). The `+` fix above should sort it out.